### PR TITLE
{data}[foss/2019b] NLTK v3.5 v/ Python 3.7.4

### PR DESCRIPTION
**Pull request by levlafayette**
Versions 3.5+ of NLTK no longer accept 2.x series Python.

### CODE REVIEW

File: easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+sources = [SOURCELOWER_ZIP]
+checksums = ['845365449cd8c5f9731f7cb9f8bd6fd0767553b9d53af9eb1b3abf7700936b35']
+
+dependencies = [('Python', '3.7.4')]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```
Comment (Micket) [inline easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb:22]: This is the default for PythonPackage, you can drop it
```suggestion
```

File: easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description = "NLTK is a leading platform for building Python programs to work with human language data."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_ZIP]
+checksums = ['845365449cd8c5f9731f7cb9f8bd6fd0767553b9d53af9eb1b3abf7700936b35']
+
+dependencies = [('Python', '3.7.4')]
+
```
Comment (smoors) [inline easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb:17]: @levlafayette please add the following lines (you might have to add extensions if the build fails with this):
```
download_dep_fail = True
use_pip = True
sanity_pip_check = True
```

**Review by smoors — DISMISSED** [review]
@levlafayette easyconfigs should always start with the software name:
NLTK-3.5-GCC-8.3.0-Python-3.7.4.eb

File: easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'PythonPackage'
+
+name = 'NLTK'
+version = '3.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.nltk.org/'
```
[the remark this answers is not in the document]
Comment (levlafayette) [reply easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb:7]: Oops, my bad. Sorry about that,

**Comment by Micket** [comment]
@levlafayette Filename is still wrong; missing NLTK

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/b3bc231baefe3f19701fa02b465d5669 for a full test report.

**Comment by Micket** [comment]
So, this package actually depends on a package called "regex" which isn't installed. This would need to become a PythonBundle that includes those

In fact, there is a whole list of dependencies; this should depend on SciPy-bundle, matplotlib, scikit-learn, 
https://github.com/nltk/nltk/blob/develop/pip-req.txt
I'm not sure how strict those actually are, but, definitely can't even be imported without `regex`

**Review by Micket — DISMISSED** [review]
See comment on dependencies

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3545.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/0865a10e5290573fe7ba504d40c2add7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3106.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/159f5828ed3f21635f72c77b2e282d64 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2608.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1c24b350e556962957195021bf6ada12 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @levlafayette!

**Pull request merged**